src/files/client-key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
All these /src/files are for testing, right?
Perhaps we could move them out of src directory. For example to /testingFixtures/certificates  ?

Also, do you use both files and files2?

Reply:
FILES contains pems from it-ws tests, thay always pass, 
and FILES2 cotains pems that are generated each time the test is running

Reply:
well, I'm not sure we need it-ws tests. This is external library after all.

Files that are generated by the tests itself, should *not* be added to the repository. They should be blacklisted in .gitignore